webui/react/src/ioTypes.ts

```diff
@@ -207,6 +207,7 @@ export const ioCheckpointStorage = io.type({
     io.literal('aws'),
     io.literal('gcs'),
     io.literal('hdfs'),
+    io.literal('s3'),
```

Comment:
can we generate this type from the `CheckpointStorageType` enum the same way we treat `runState` . This should avoid these two going out of sync

Reply:
great point! updated